docs/tsg.md

```diff
@@ -231,8 +232,27 @@ Client connections can drop under various circumstances:
 2. Check app server side event log to see if the app server restarted
 3. Create an issue to us providing time frame, and email the resource name to us
 
+<a name="client_connection_increases_constantly"></a>
+## Client connection increases constantly
+It might be caused by improper usage of client connection. If someone forgets to stop/dispose SignalR client, the connection remains open. This issue often occurs when someone establishs SignalR client connection in Azure Function method instead of making it a static member to your Function class. You might expect only one client connection is established, but you see client connection count increases constantly in metrics blade, all this connections drop only after the Azure Function or Azure SignalR service restarts. This is because for **each** request, Azure Function creates **one** client connection, if you don't stop client connection in Function method, the client keeps the connections alive to Azure SignalR service.
+
+### Possible errors seen from the SignalR's metrics blade
+Client connections rise constantly for a long time in Azure SignalR's metrics blade.
+![client_connection_increasing_constantly](./images/client_connection_increasing_constantly.jpg)
+
+### Root cause:
+SignalR client connection's `StopAsync` or `DisposeAsync` never be called, the connection keeps open.
+
+### Troubleshooting Guide
+1. Check if SignalR client used in Azure Function.
+1. Check if the SignalR client **never** stop.
+
+### Solution
+1. Check if you design a proper serverless architecture. You can refer to [Real-time serverless applications with the SignalR Service bindings in Azure Functions](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.SignalRService).
```

Comment:
The first solution we give to customer should be calling `StopAsync` in his code?